test/unit/options/normalizers_test.rb

```diff
@@ -429,15 +439,45 @@
     end
 
     it "should accept and parse iso8601" do
-      _(formatter.format("1986-01-01T08:30:20")).must_equal("1986-01-01T08:30:20+00:00")
+      mock_env("TZ" => "UTC") do
+        _(formatter.format("1986-01-01T08:30:20")).must_equal("1986-01-01T08:30:20+00:00")
+      end
     end
 
     it "should accept and parse YYYY-MM-DD HH:MM:SS" do
-      _(formatter.format("1986-01-01 08:30:20")).must_equal("1986-01-01T08:30:20+00:00")
+      mock_env("TZ" => "UTC") do
+        _(formatter.format("1986-01-01 08:30:20")).must_equal("1986-01-01T08:30:20+00:00")
+      end
     end
 
     it "should accept and parse YYYY/MM/DD HH:MM:SS" do
-      _(formatter.format("1986/01/01 08:30:20")).must_equal("1986-01-01T08:30:20+00:00")
+      mock_env("TZ" => "UTC") do
+        _(formatter.format("1986/01/01 08:30:20")).must_equal("1986-01-01T08:30:20+00:00")
+      end
+    end
+
+    it "should parse iso8601 compliant timezones" do
+      mock_env("TZ" => "UTC") do
+        _(formatter.format("1986-01-01 08:30:20 +04:00")).must_equal("1986-01-01T08:30:20+04:00")
+      end
+    end
+
+    it "should not recognize invalid timezones" do
+      mock_env("TZ" => "UTC") do
+        _(formatter.format("1986-01-01 08:30:20 bullshit")).must_equal("1986-01-01T08:30:20+00:00")
+      end
```

Comment:
Could you either duplicate this one or change it from utc to something else to be extra clear it's picking up the timezone from the environment and not silently defaulting to utc?